Sources/Services/AWSPolly/PollyClient.swift

```diff
@@ -481,3 +484,37 @@ extension PollyClient: PollyClientProtocol {
     }
 
 }
+
+extension PollyClient {
+    /// Presigns the URL for SynthesizeSpeech operation with the given input object SynthesizeSpeechInput.
```

Comment:
The doc comment should include the same docs that are rendered onto the regular operation.

Reply:
I don't understand what you mean by this - do you mean the links to AWS docs (the ones in Smithy model under @documentation trait)? If so, they aren't available for these presign operations because these are added at SDK level.

Reply:
Discussed offline - will add docs as mentioned.